dockers/broad/verify_bam_id/README.md

```diff
@@ -0,0 +1,44 @@
+# VerifyBamID
+
+## Quick reference
+
+Copy and paste to pull this image
+
+#### `docker pull us.gcr.io/broad-gotc-prod/verify-bam-id:1.0.0-c1cba76e979904eb69c31520a0d7f5be63c72253-1626442707`
+
+- __What is this image:__ This image is a debian-based image for running the VerifyBamID tool.
+- __What is VerifyBamID:__ VerifyBamId is a tool for DNA contamination estimation from sequence reads using ancestry-agnostic method. See [here](https://github.com/Griffan/VerifyBamID) more information.
+- __How to see VerifyBamID version used in image:__ Please see below.
+
+## Versioning
+
+The VerifyBamID image uses the following convention for versioning:
+
+#### `us.gcr.io/broad-gotc-prod/samtools:<image-version>-<VerifyBamID-git-hash>-<unix-timestamp>` 
```

Comment:
I believe this should be `us.gcr.io/broad-gotc-prod/verify-bam-id` instead of `us.gcr.io/broad-gotc-prod/samtools`